plugins/chain/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ target_link_libraries(
         fc
         appbase
         golos::json_rpc
+        golos::follow
```

Comment:
is it really required?

Reply:
agree. removing this dep allows build state serializer without follow plugin at all. removed